Util/MeshUtil.cs

```diff
@@ -248,12 +248,83 @@ public static string RenderableListToObjFile(List<IRenderable> meshes)
             return objString.ToString();
         }
 
+
+        public static string RenderableToPlyFile(IRenderable mesh)
+        {
+            int vertCount = mesh.Verts.Vertices.Length;
+            int triCount = mesh.Verts.Tris;
+
+            PlyBuilder plyString = new PlyBuilder();
+            plyString.WriteHeader(vertCount, triCount);
+
```

Comment:
You can fix the rotation issue with the following code here:
```c#
Matrix4 rotX90Deg = Matrix4.CreateRotationX(MathHelper.DegreesToRadians(-90));

// Vertex elements
foreach (var vert in mesh.Verts.Vertices)
{
    plyString.BuildVertexElement(Vector3.TransformPosition(vert.Position, mesh.Transform.Matrix * rotX90Deg), vert.Normal, vert.TexCoord, vert.Color);
}
```

PLY files must have a slightly different coordinate space, but rotating it by -90 degrees on X seems to fix that :)